WireDomain/Tests/WireDomainTests/Event Processing/UserEventProcessor/UserLegalHoldDisableEventProcessorTests.swift

```diff
@@ -0,0 +1,105 @@
+//
+// Wire
+// Copyright (C) 2024 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import Foundation
+import WireAPI
+import WireAPISupport
+import WireDataModel
+import WireDataModelSupport
+import XCTest
+
+@testable import WireDomain
+
+final class UserLegalHoldDisableEventProcessorTests: XCTestCase {
+
+    var sut: UserLegalholdDisableEventProcessor!
+
+    var coreDataStack: CoreDataStack!
+    let coreDataStackHelper = CoreDataStackHelper()
+    let modelHelper = ModelHelper()
```

Comment:
Personally I'm pro creating and destroying these in setup/teardown because:

1. They are kept in memory until all unit tests complete. They might be lightweight in this case but it is easier to have a general process so we don't need to check this on a case by case basis.
2. `CoreDataStackHelper` is fine like this when it's in memory but if someone changed it to on disk, tests can start interfering with each other.

Reply:
done thx